### PR TITLE
Reformatted Entry

### PR DESCRIPTION
**Pull request by ACMClassicalCode**
Reformatted entries/Alkatraz/Alkatraz.md to the best of my ability.

### CODE REVIEW

File: entries/Alcatraz/Alcatraz.md [hunk 1]
```diff
@@ -1,44 +1,129 @@
...
+## Names: 
+
+* Alcatraz[^Alcatraz_Product_Page_2003], Alkatraz[^Alkatraz_Usage]
+
+* [^Alcatraz_Product_Page_2003]: [Alcatraz product page. (Archived 2003-10-11)](https://web.archive.org/web/20031011203913/http://www.kdg-mt.com/1/product_d_6.html)
+
+* [^Alkatraz_Usage]: Unofficial/Incorrect name used in a few places, such as a forum post about CD copy protections[^Alkatraz_indiastudychannel] and a journal paper about DRM.[^Alkatraz_Arxiv] It's worth noting that "Alkatraz" is also the proper name of a copy protection for the Amstrad CPC.[^Alkatraz_Amstrad]
 
-[^Alkatraz_indiastudychannel]: [Indiastudychannel forum post that briefly mentions "Alkatraz". (Archived 2023-10-02)](https://web.archive.org/web/20231002194853/https://www.indiastudychannel.com/experts/4307-Copy-protected-cd)
+* [^Alkatraz_indiastudychannel]: [Indiastudychannel forum post that briefly mentions "Alkatraz". (Archived 2023-10-02)](https://web.archive.org/web/20231002194853/https://www.indiastudychannel.com/experts/4307-Copy-protected-cd)
```
Comment (TheRogueArchivist) [inline entries/Alcatraz/Alcatraz.md]: The footnotes shouldn't be made into a list, as it causes the Markdown to be rendered with several blank list items.

File: entries/Alcatraz/Alcatraz.md [hunk 2]
```diff
@@ -135,3 +231,4 @@ There were some informational PDFs present on the site, which were removed at th
 
 ### Footnotes:
 
+{This section should be completely empty in the editor. This section will be automatically populated by the footnotes created throughout the document, which should be created at the end of the section they are first used in. Names for each footnote should be sufficiently descriptive to clearly indicate what footnote they're referring to in the document to avoid confusion. These names don't need to be overly descriptive, but it's better to err on the side of being too descriptive instead of not descriptive enough.}
```
Comment (TheRogueArchivist) [inline entries/Alcatraz/Alcatraz.md]: This section was accidentally left in from the entry template.

File: entries/Alcatraz/Alcatraz.md [hunk 3]
```diff
@@ -77,33 +162,44 @@ Any mention of these additional protection levels was removed from the product p
...
+* {Unordered list of platforms this entry's DRM is supported on. This should include specific versions or products, if known.}
 
-There were some informational PDFs present on the site, which were removed at the same time as the reference to multiple protection levels was removed. Unfortunately, only one of these appears to have been archived.[^Alcatraz_Promo]
+***
 
-[TODO: Find older official material, as the only official material currently known is from 2003 onward.]
+## Associated File Attributes:
 
-[^Alcatraz_Promo]: [Alcatraz promotional material. (Archived 2004-04-16)](https://web.archive.org/web/20040416164237/http://www.kdg-mt.com/downloads/1/al.pdf)
+### {Series of level 3 headers used to denote what attribute this entry's DRM has which can be used to identify it. TODO: Make standard list of file attributes that may be used.}
```
Comment (TheRogueArchivist) [inline entries/Alcatraz/Alcatraz.md]: Some entry template text seems to have been left in here.

**Comment by ACMClassicalCode** [comment]
Still has the Versions test issue. Send some feedback on how it should be done, if you would like me to fix it.

**Comment by TheRogueArchivist** [comment]
It seems that most of my issues have been addressed, but it seems you made the changes to "alkatraz.md" instead of "alcatraz.md". If you could remove the "alkatraz.md" file, and move the changes into "alcatraz.md", please! Also it looks like the footnotes in the "Names" sections are still marked as part of a list, if you could remove the asterisks from the start of the footnotes there to fix the list that should fix the issue.

Also which "versions test issue"? If you mean the descriptions of the protection levels, that's no problem at all and I can handle those myself later!

**Comment by ACMClassicalCode** [comment]
Thank you for being clear about what you needed done. Hopefully this is the last commit :)

**Review by TheRogueArchivist — APPROVED** [review]

**Comment by TheRogueArchivist** [comment]
Looks good to me, and thank you very much for your contribution! 🎉

**Pull request merged**